app/ai/voice/agents/breeze_buddy/memory/service.py

```
@@ -0,0 +1,126 @@
+            from app.database.accessor.breeze_buddy.user_memory import (
+                list_user_memories,
+            )
+
+            memories = await list_user_memories(reseller_id, merchant_id, customer_key)
+            if not memories:
+                return None
+
+            lines = [m.fact for m in memories[:max_facts]]
+            body = "\n".join(f"- {line}" for line in lines)
+            return f"<user_memory>\n{body}\n</user_memory>"
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)